hardhat.config.ts

```diff
@@ -95,6 +98,18 @@ task('allowed-tokens', 'Retrieves a list of allowed tokens')
     await getAllowedTokens(hre);
   }
 );
+task('withdraw', 'Retrieves a list of allowed tokens')
+  .addParam('collectorAddress', 'address of the collector we want to withdraw from')
+  .addParam('partnerConfig', 'path of the file that includes the partner shares configuration')
+  .addOptionalParam('gasLimit', 'gasLimit to be used for the transaction')
+  .setAction(async (taskArgs: { 
+    collectorAddress: string,
+    partnerConfig: string,
+    gasLimit?: BigNumberish
+  }, hre) => {
```

Comment:
I'd use the type exported from `scrits/withdraw.ts`

Reply:
done